lib/chef/application.rb

```diff
@@ -151,6 +152,20 @@ def configure_logging
       Chef::Application.fatal!("Aborting due to invalid 'log_location' configuration", 2)
     end
 
+    # Turn `log_location :syslog` and `log_location :win_evt` into the
+    # appropriate loggers.
+    def configure_log_location
+      log_location = Chef::Config[:log_location]
+      return unless log_location.respond_to?(:to_sym)
+
+      Chef::Config[:log_location] =
+        case log_location.to_sym
```

Comment:
Same question: if we're not intending to allow something like `log_location "win_evt"`, I feel like we should reject it.